model-optimizer/mo/utils/ir_reader/extenders/pooling_extender.py

```diff
@@ -50,7 +52,7 @@ def common_pool_extender(op: Node):
 
     dim = len(op.pads_begin)
 
-    assert dim in (2, 3), '{}D {} not supported!'.format(dim, op.op)
+    assert dim in (1, 2, 3), '{}D {} not supported!'.format(dim, op.soft_get('name', op.id))
```

Comment:
You have replaced operation type by node name in error message but I think both are useful for debug. Could you add type too?

Reply:
I think the same that `type` should be here.

Reply:
Yes, you are right, but this function used only for Pooling operations only in MO IR Reader and in this case operation name more necessary then type. I think we don't need them both, because operation from this message can be easily found in IR by name.

Reply:
But ok, i update this message